src/lwc/geReviewDonations/geReviewDonations.js

```diff
@@ -177,4 +176,9 @@ export default class geReviewDonations extends NavigationMixin(LightningElement)
                 handleError(error);
             });
     }
+
+    @api
+    setProperty(property, value) {
```

Comment:
This gives me pause. Do we need to be able to modify every property on this component?

Reply:
updated!